lib/test.gi

```diff
@@ -114,6 +118,13 @@ InstallGlobalFunction(RunTests, function(arg)
   breakOnError := BreakOnError;
   BreakOnError := opts.breakOnError;
 
+  localbag := false;
+  if opts.localdef <> false then
+    # This makes a local variables bag by creating a new function and 
+    # calling EvalString
+    localbag := EvalString(Concatenation("(function() local ", opts.localdef,
+                                         "; return GetCurrentLVars(); end)()"));
```

Comment:
We could of course write a dedicated function to replace this `EvalString` hack. But I don't actually think it would have any benefit, other than perhaps "feeling cleaner".